Dockerfile.template

```diff
@@ -1,13 +1,13 @@
 # https://nodejs.org/en/about/releases/
 # https://github.com/nodejs/Release#readme
-FROM node:18-alpine3.16
+FROM node:{{ env.variant }}
 
 # override some config defaults with values that will work better for docker
 ENV ME_CONFIG_MONGODB_URL="mongodb://mongo:27017" \
     ME_CONFIG_MONGODB_ENABLE_ADMIN="true" \
     VCAP_APP_HOST="0.0.0.0"
 
-ARG MONGO_EXPRESS_VERSION=1.0.0
+ENV MONGO_EXPRESS_VERSION={{ .version }}
 
 COPY docker-entrypoint.sh /
```

Comment:
This `COPY` should be after the `RUN` (better build caching) and `docker-entrypoint.sh` should be `chmod +x` in git rather than adjusted in the Dockerfile (otherwise it is duplicated in the image).

Reply:
Created #94